### PR TITLE
Rename Quality control and data team helper to imaging/behavioural QC and reorganise menu

### PR DESCRIPTION
**Pull request by ridz1208**
## Brief summary of changes
- removed (hopefully) all reference to "data team helper" and replaced with Behavioural Quality Control
- removed bahavioural tab from QC module and renamed the module to Imaging Quality Control
- reoganised menu
  - Imaging QC -> imaging tab
  - Behavioural QC -> clinical tab
  - genomic browser -> (new) genomics tab

Full module tests should be run and code grepped to ensure nothing is left behind

### CODE REVIEW

**Comment by maltheism** [comment]
I'm referencing https://github.com/aces/Loris/pull/5330 and to help me refactor more easily.

**Comment by ridz1208** [comment]
@maltheism sorry about this... orders from above

**Comment by christinerogers** [comment]
I think the top-line menu change should have been issued separately because it's a big change in the cosmetics of the top menu bar.

Tagging `needs documentation`
There's a Readme coming for this module from @knoll-alex - fyi @LeighMac  - see [Alex's PR on her fork](https://github.com/knoll-alex/Loris/pull/1) which will be updated accordingly and rebased

File: modules/behavioural_qc/test/behavioural_qcTest.php [hunk 1]
```diff
@@ -62,10 +62,10 @@ function testDataTeamHelperWithoutPermission()
      *
      * @return void
      */
-    function testDataTeamHelperPermission()
+    function testBeahaviouralQCPermission()
```
Comment (christinerogers) [inline modules/behavioural_qc/test/behavioural_qcTest.php]: typo : `Beahav`

**Pull request merged**